packages/sdk/src/types/runtime/behaviors/action.ts

```diff
@@ -52,7 +52,7 @@ export class DiscreteAction {
 				targetId: actorId,
 				actionName: this.name,
 				actionState,
-				action: options.action,
+				triggeredAction: options.action,

```

Comment:
Should the naming here be options.triggeredAction here as well for consistency?

Reply:
Yes, good catch.

---
In reply to: [316894771](https://github.com/microsoft/mixed-reality-extension-sdk/pull/425#discussion_r316894771) [](ancestors = 316894771)

Reply:
Renamed.

---
In reply to: [316896105](https://github.com/microsoft/mixed-reality-extension-sdk/pull/425#discussion_r316896105) [](ancestors = 316896105,316894771)